intermediate-week-1/utils/accessToken.js

```diff
@@ -0,0 +1,17 @@
+require('dotenv').config()
+const jwt = require('jsonwebtoken')
+const {JWT_REFRESH_SECRET, JWT_EXPIRES, JWT_SECRET, JWT_REFRESH_EXPIRES} = process.env
+// const User = require('../../models/User')
+
+// const errorHandler = require('../../error/errorHandler')
+
+exports.refreshAccessToken1 =  (user) => {
+    
+   return jwt.sign({user}, JWT_REFRESH_SECRET, {expiresIn: JWT_REFRESH_EXPIRES})
```

Comment:
Well done. NIce work here. Seperating the login of the access and refresh token into another file to make it more readable. 

Well done.